README.md

```diff
@@ -1,2 +1,46 @@
-# HelloMicroverse
-This is hello Microverse project ( Module 1 )
+📖 Hello Microverse project (Module 1)
+
+This is my HELLO MICROVERSE Project from the module one.
+
+🛠 Built With
+
+HTML
+Linters.
+GitHub/VSCode.
+🚀 Live Demo
+
+Soon
+💻 Getting Started
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+
+Internet connection.
+Github.
+VScode / Atom / sublime text or any text editor ready for programming languages.
+Internet browser (GOOGLE Chrome recommanded).
+Setup Clone this repository to your desired folder.
+
+👤 Author
+
+GitHub: @Aminocodex
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+(back to top)
+
+⭐️ Show your support Write a message to encourage readers to support your project
+
+If you like this project, give it a ⭐️.
+
+(back to top)
+
+🙏 Acknowledgments Give credit to everyone who inspired your codebase.
+
+a big thanks for anyone whose code was used to make this project operational
+(back to top)
+
+📝 License This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
Hello, @Aminocodex I noticed that you didn't properly implement the previous changes I suggested you implemet so I will try to elaborate further so you can have a clear idea :+1: 

- [ ] Firstly, the README file is still not well done. I am going to give you a step-by-step process. Firstly, copy the [Microverse README template](https://github.com/microverseinc/readme-template) into your README then you can make these changes to it accordingly in order to customize it to suit your project :100: 

- [ ] Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 

- [ ] For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS.

- [ ] For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file.

- [ ] For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        ◦ Installation of Git
        ◦ Installation of NPM
        ◦ A working browser
        ◦ Internet connection
        ◦ A code editor like Visual Studio Code
        ◦ Installation of Live Server plugging in Visual Studio Code
        ◦ Any other tool you can think about

- [ ] For the Setup section, kindly add instructions for cloning your repository on a local machine.

- [ ] For the Install section, kindly add instructions for setting up linters used in this project

- [ ] For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.

- [ ] For the MIT section, you can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 

- [ ] Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.  

Reply:
This same template of read me file i did with another reviewer and he guided me a step by step to do it, Can you please point where is the issue exactly in this readme file ?

Reply:
Can you please let me know where do you see the part the Server and Database subsections  ? Can you point up on which line do you see that ? 
I created the read me file with another reviewer for the portfolio project, Can you please let me know where is the problem with this one exactly? 
Another thing, The project requirement didn’t ask to create a Licence.md file. It is only required to be put in the readme file. Can you check again if I really need to create one ?? 
Looking forward for your reply !